pkgs/development/python-modules/rubicon-objc/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, darwin
+, fetchFromGitHub
+, pythonOlder
+, setuptools
+, setuptools-scm
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "rubicon-objc";
+  version = "0.4.8";
```

Comment:
Can you update the commit message to match the version number?

Reply:
Good catch, thanks!